db/migrate/20201029164928_add_theme_to_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddThemeToUsers < ActiveRecord::Migration[6.0]
+  def change
+    add_column :users, :theme, :string, default: 'light', null: false
```

Comment:
Let's make this new column an Enum: https://edgeapi.rubyonrails.org/classes/ActiveRecord/Enum.html.